libc/src/__support/FPUtil/FloatProperties.h

```diff
@@ -162,13 +165,20 @@ struct FPProperties : public internal::FPBaseProperties<fp_type> {
       FRACTION_LEN + 1;
   LIBC_INLINE_VAR static constexpr StorageType FRACTION_MASK =
       mask_trailing_ones<StorageType, FRACTION_LEN>();
-  LIBC_INLINE_VAR static constexpr StorageType EXP_MANT_MASK =
-      EXP_MASK | SIG_MASK;
 
   // If a number x is a NAN, then it is a quiet NAN if:
   //   QuietNaNMask & bits(x) != 0
   // Else, it is a signalling NAN.
   static constexpr StorageType QUIET_NAN_MASK = QNAN_MASK;
+
+  //---------------------------------------------------------------------------
+  // Modifiers
+  //---------------------------------------------------------------------------
+
+  // Returns the absolute value of 'value'.
+  LIBC_INLINE static constexpr StorageType abs(StorageType value) {
```

Comment:
It's weird to have a member function in a class named `FloatProperties`. This should probably be a free function with template properties.

Actually, I'm noting that this is aways called on the result of calling `FPBits::uintval()`, maybe it's a sign that this should be a member function of `FPBits` ?

Reply:
Fair enough, I wanted to unify `FloatProperties` and `FPBits` at some point but it's probably better to do this afterwards. It is clear that `FPBits` is a better place for `abs` right now. Thx for the suggestion.